style/typescript/.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "arrowParens": "always",
```

Comment:
I believe that prettier 2.0 makes this the default.

https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev

Reply:
Thanks for catching this 🙂 

I removed this over in https://github.com/thoughtbot/react-suspenders/pull/1/commits/b702b5ec3ea86675365cc0068ec2e84c94e80209